styles/headline-section.css

```diff
@@ -1,21 +1,122 @@
-.mobile-nav {
+header {
   height: 72px;
   background-color: #3e3c3c;
+}
+
+.mobile-nav {
+  background-color: #3e3c3c;
+  display: flex;
+  justify-content: space-between;
   position: fixed;
   width: 400px;
   top: 0;
-  z-index: 3;
+  z-index: 4;
+}
+
+.hamburger {
+  cursor: pointer;
+}
+
+.bar {
+  display: block;
+  top: 10px;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all ease-in-out;
+  background-color: white;
+  border-radius: 10px;
+  margin-right: 30px;
+}
+
+.hamburger-X {
+  cursor: pointer;
+  display: none;
+}
+
+.bar:nth-child(1) {
+  margin-top: 20px;
+}
+
+.hamburger-X .bar:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+}
+
+.hamburger-X .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburger-X .bar:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+}
+
+.hamburger:active .bar:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+}
+
+.hamburger:active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburger:active .bar:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
 }
 
-.display-nav {
+.nav-ul {
+  display: none;
+  position: fixed;
+  top: 50px;
+  left: 0;
+  gap: 0;
+  flex-direction: column;
+  background-color: #3e3c3c;
+  list-style-type: none;
+  vertical-align: center;
+  color: white;
+  padding: 10px;
+  margin: 0;
+  width: 100%;
+  align-items: center;
+  text-align: center;
+}
+
+.nav-ul:active {
+  left: 0;
+  display: flex;
```

Comment:
In this exercise the goal is to replicate the figma design as much as possible. However, I can see that there are some differences between your design and that of the figma design as indicated below. :point_down: 
Kindly ensure that your design matches that of the figma design in terms of the length, font-size, margins, paddings, etc

### Your design

![Screenshot from 2023-01-03 16-57-46](https://user-images.githubusercontent.com/65233860/210409074-15a0ab7a-ece6-4384-8a98-94e0f04b62fc.png)


### Figma design 

![Screenshot from 2023-01-03 16-57-34](https://user-images.githubusercontent.com/65233860/210409110-d55743df-5616-4ded-9a26-63fe1d93fa4e.png)